### ISSUE TITLE
ruler: prometheus_rule_group_last_duration_seconds is not working.

### ISSUE THREAD

**Issue by bwplotka** [thanos-io/thanos#1027]
I can see the values out of the world totally. Is it even updated? Let's fix this.

**Comment by yeya24** [comment]
In prometheus 2.7.1, this metric used seconds instead of nanoseconds.
Here is the PR [#5153](https://github.com/prometheus/prometheus/pull/5153#issue-248761847).
And in thanos, it is still the old package using nanoseconds.
``` go
func (g *Group) setEvaluationDuration(dur time.Duration) {
	g.metrics.groupLastDuration.WithLabelValues(groupKey(g.file, g.name)).Set(float64(dur))

	g.mtx.Lock()
	defer g.mtx.Unlock()
	g.evaluationDuration = dur
}
```

**Comment by bwplotka** [comment]
Wow, thanks for this, that might be it. But still it's bit off

**Comment by GiedriusS** [comment]
Was this fixed after the library has been updated? 😄

**Comment by yeya24** [comment]
Is this issue fixed now?

**Comment by jojohappy** [comment]
@yeya24 It seems to be fixed in #1009 with v0.4.0. Let's double check it.

**Comment by simonpasquier** [comment]
Using master, I see that `prometheus_rule_group_last_duration_seconds` values match with the rules UI.
